### PR TITLE
data:Labhk

### PR DESCRIPTION
**Pull request by Labhk**

### CODE REVIEW

File: data/Labhk.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Labh Khatke",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Currently a College student | Web Developer | Open Source Enthusiast",
+  "avatar": "https://i.ibb.co/XyWdRrF/Me.jpg",
```
Comment (Aadarsh805) [inline data/Labhk.json:6]: Change avatar link to `https://github.com/Labhk.png`

File: data/Labhk.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/labh_k77",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/labh-khatke-4436b0180",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline data/Labhk.json:26]: change this LinkedIn icon, use `FaLinkedin`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 👍

**Pull request merged**